pkg/gateway/operations/putobject.go

```diff
@@ -325,3 +338,20 @@ func handlePut(w http.ResponseWriter, req *http.Request, o *PathOperation) {
 	o.SetHeader(w, "ETag", httputil.ETag(blob.Checksum))
 	w.WriteHeader(http.StatusOK)
 }
+
+func (o *PathOperation) checkIfAbsent(req *http.Request) (bool, error) {
+	Header := req.Header.Get(IfNoneMatchHeader)
```

Comment:
- Use lowercase for variable names 
- Be explicit in variable name

```suggestion
	headerValue := req.Header.Get(IfNoneMatchHeader)
```